### PR TITLE
Chatbot - Clean up from code

### LINKED ISSUE #1502: 💸 Tech Debt - Integrate chatbot via Google Tag manager
Subject: RE: Urgent - GTM tracking for SSW.fr may be missing
CC: @JeanThirion @tiagov8 @camillars @sethdailyssw @divgul 

**Description**  
Currently, we are manually adding the chatbot to our code. Ideally, it should be managed through Google Tag Manager (GTM) since all our scripts are generated through GTM. Centralizing it in GTM would streamline maintenance and management.

```[tasklist]
### Tasks
- [x] ~Add  Chatbot script as a tag in the GTM~
- [x] ~Clean up the Chatbot scripts from the code~
- [x] ~Doc - Add GTM tag for any third party scripts~
```

### PR DESCRIPTION
**Pull request by amankumarrr**
- Related #1502 

**Description:**
Removing Chatbot from the code as it's being added by Google tag manager as a tag

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -12,7 +12,6 @@ import { NEXT_SEO_DEFAULT } from "../next-seo.config";
 import "../styles.css";
 
 import AzureAppInsights from "../context/app-insight-client";
-const chatBaseBotId = process.env.NEXT_PUBLIC_CHATBASE_BOT_ID;
```
Comment (wicksipedia) [inline pages/_app.tsx:15]: do we need to remove this from the `.env.example` file and keeper?

**Comment by wicksipedia** [comment]
when I go to the PR site - there is no chatbot... is that expected?

CleanShot 2023-10-10 at 12 48 43@2x

**Comment by amankumarrr** [comment]
> when I go to the PR site - there is no chatbot... is that expected?
> 
> CleanShot 2023-10-10 at 12 48 43@2x

No, it takes a while to load it for the first time, can you try it reloading the page?

File: pages/_app.tsx (hunk 1 above)
Comment (amankumarrr) [reply pages/_app.tsx:15]: I removed NEXT_PUBLIC_ZENDESK_CHAT_KEY from `.env.example` and `NEXT_PUBLIC_CHATBASE_BOT_ID` from our `.github/.env`, we don't have these keys in keeper

**Comment by wicksipedia** [comment]
> No, it takes a while to load it for the first time, can you try it reloading the page?

I stayed on <https://app-sswwebsite-9eb3-pr-1516.azurewebsites.net/> for 30 seconds and it did not load

**Comment by wicksipedia** [comment]
looks like it is being blocked by my ad blocker - https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm

i'd recommend not going ahead with this change as users with ad blockers won't see this cool feature

**Comment by amankumarrr** [comment]
Closing this issue as it may be blocked by ad blockers, and therefore, this change will not progress further.

**Pull request closed without merging**